src/query/settings/src/settings_default.rs

```diff
@@ -510,6 +510,18 @@ impl DefaultSettings {
                     mode: SettingMode::Both,
                     range: Some(SettingRange::Numeric(0..=1)),
                 }),
+                ("fuse_write_use_parquet2", DefaultSettingValue {
+                    value: UserSettingValue::UInt64(0),
+                    desc: "Use parquet2 instead of parquet_rs when writing data with fuse engine.",
+                    mode: SettingMode::Both,
+                    range: Some(SettingRange::Numeric(0..=1)),
+                }),
+                ("fuse_read_use_parquet2", DefaultSettingValue {
```

Comment:
We should use `fuse_use_parquet2` setting instead `fuse_write_use_parquet2` and `fuse_read_use_parquet2`, as if `fuse_write_use_parquet2` and `fuse_read_use_parquet2` are not same may lead to unintended results.

Reply:
👍, now the two settings are very convenient for testing. Let's merge them later.